docker-compose.yml

```diff
@@ -2,16 +2,45 @@ version: '3.8'
 services:
   mysql:
     image: mysql:8.0
-    command: --default-authentication-plugin=mysql_native_password
+    restart: always
+    command:
+      --default-authentication-plugin=mysql_native_password
+      --bind-address=0.0.0.0
     cap_add:
       - SYS_NICE
     ports:
-      - '3306:3306'
+      - 3306:3306
     environment:
```

Comment:
better to use `.env` file for specifying environment variables. It could give more flexibility in the future:
```yml
    env_file:
      - .env
```